upup/pkg/fi/nodeup/nodetasks/package.go

```diff
@@ -162,7 +169,7 @@ func (e *Package) findDpkg(c *fi.Context) (*Package, error) {
 			installed = true
 			installedVersion = version
 			healthy = fi.Bool(true)
-		case "iF":
+		case "iF", "iU":
```

Comment:
What’s this iU?

Reply:
This happened in an earlier version of this change that attempted to install the packages one by one (which, ultimately, doesn't work).  `iF` means the package is selected to be installed and is currently in half-configured state, while `iU` means it's selected to be installed and is currently merely unpacked.

It might not be needed anymore since this iteration of the code installs all 3 packages in a single transaction, but having the code craps out if it sees the package in `iU` state instead of continuing to try/wait until the install is complete seems like a bad idea.  So I figured I'd leave this here as it makes sense to handle `iU` in the same way that `iF` is handled.